src/util/fees.cpp

```diff
@@ -26,16 +29,32 @@ std::string StringForFeeReason(FeeReason reason) {
     return reason_string->second;
 }
 
-bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_estimate_mode) {
-    static const std::map<std::string, FeeEstimateMode> fee_modes = {
+const std::vector<std::pair<std::string, FeeEstimateMode>>& FeeModeMap() {
```

Comment:
457396a4091bb3c27b1e982fb7e08442120b15df

Why have this as a function? Could be a static variable in this translation unit, like
```cpp
static const std::vector<std::pair<std::string, FeeEstimateMode>> FEE_MODES{
    {"UNSET", FeeEstimateMode::UNSET},
    {"ECONOMICAL", FeeEstimateMode::ECONOMICAL},
    {"CONSERVATIVE", FeeEstimateMode::CONSERVATIVE},
    {(CURRENCY_UNIT + "/KB"), FeeEstimateMode::BTC_KB},
    {(ToUpper(CURRENCY_ATOM) + "/B"), FeeEstimateMode::SAT_B},
};
```

Reply:
Windows build breaks if I do this, unfortunately.

Reply:
Now that `ToUpper` is no longer used, this may actually work in Windows; tentatively removing the function wrapper.